testing/efl/APKBUILD

```diff
@@ -0,0 +1,72 @@
+# Contributor: Boris Faure <boris@fau.re>
+# Maintainer:
```

Comment:
The package should have a maintainer. Do you want to maintain this package?

Reply:
Yes, I do.